laserscan_kinect/include/laserscan_kinect/laserscan_kinect.h

```diff
@@ -175,6 +175,10 @@ class LaserScanKinect
         float depth_min = std::numeric_limits<float>::max();
         const unsigned range_min_mm = range_min_ * 1000;
         const unsigned range_max_mm = range_max_ * 1000;
+        float depth_prev_prev = -1000;
```

Comment:
weird variable name...not quite sure what it's purpose is based of the name. could probably be named better and/or use comments to describe use